ovs/lib/vdisk.py

```diff
@@ -1588,7 +1589,7 @@ def clean_devicename(name):
         :return: A cleaned device name
         :rtype: str
         """
-        name = name.strip('/').replace(' ', '_')
+        name = str(name).strip('/').replace(' ', '_')
```

Comment:
Clean device name expects a string, so better to check if the given item is a string rather than stringifying it yourself